frontend/src/clients/externalApiClient.ts

```diff
@@ -1,53 +1,81 @@
-export interface Sightings {
-  sightings: Sighting[];
+export interface SightingsListResponse {
```

Comment:
Close! 😉 
```suggestion
export interface SightingListResponse {
```
It's standard to have it as a singular if it's before ("item list", "song list") and plural if after ("list of items", "list of songs")